app/models/opportunity.rb

```diff
@@ -14,6 +14,8 @@ class Opportunity < ActiveRecord::Base
   has_one :sub_program, through: :voucher
 
   has_one :active_match, -> {where(active: true, closed: false)}, class_name: 'ClientOpportunityMatch'
```

Comment:
We should comment this out (deprecate it) since it shouldn't be valid anymore.  That should help find the places where we use it too.

Reply:
Commented out and made an initial pass through the code to find dependencies, replacing them with the first item in `active_matches` - the pages will need to be inspected to see if they need to be changed to display multiple active matches on an opportunity.